contributors/devel/api-conventions.md

```diff
@@ -272,7 +272,7 @@ desired state.
 
 The PUT and POST verbs on objects MUST ignore the "status" values, to avoid
 accidentally overwriting the status in read-modify-write scenarios. A `/status`
-subresource MUST be provided to enable system components to update statuses of
+subresource MUST be provided to enable system components to update statuses and metadata of
```

Comment:
there is debate over whether this is desired, especially as the amount of data in `metadata` is expanded (see https://github.com/kubernetes/kubernetes/pull/45552). I don't think status subresources should be required to expose all of metadata to modification